pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/pendingack/PendingAckPersistentTest.java

```diff
@@ -344,4 +346,42 @@ private void testDeleteTopicThenDeletePendingAckManagedLedger() throws Exception
         assertFalse(topics.contains(MLPendingAckStore.getTransactionPendingAckStoreSuffix(topic, subName2)));
         assertFalse(topics.contains(topic));
     }
+
+    @Test
+    public void testTxnCumulativeAckInOrder() throws Exception{
```

Comment:
Without this PR, the test still can get passed, could you please help check if this is a correct test for cover the new changes?